skycatalogs/utils/common_utils.py

```diff
@@ -46,6 +47,18 @@ def log_callinfo(prog, args, logname):
             log_out += '    {}: {}\n'.format(e, eval(nm))
     logger.info(log_out)
 
+def callinfo_to_dict(args):
+    """
+    Make a dict out of program arguments.  Each option value is
+    either a simple atomic type or a list
+    """
+    a_dict = dict()
+    for e in dir(args):
+        if not e.startswith('_'):
+            v = eval('args.' + e)
+            a_dict[e] = v
+    return a_dict
+
```

Comment:
It seems that
```
dict(args._get_kwargs())
```
does the same thing as this function.

Reply:
So it does.   I could also use `_get_kwargs` to simplify `log_callinfo` but the results aren't alphabetized, which is useful in that case. `print_callinfo` isn't used so I will remove it.

Reply:
To alphabetize by keyword,
```
dict(sorted(args._get_kwargs()))
```
would work.

Reply:
Ok.  I'll make that change.